# File: CustomData.msg
float64 longitude
float64 latitude
float64 altitude
string time
float64 actual_speed

## For testing
float32 time_difference
